<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>membermanager::gui::SettingsDialog</class>
 <widget class="QDialog" name="membermanager::gui::SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Bank</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="accountLabel">
        <property name="text">
         <string>Kontonummer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="accountLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="bankLabel">
        <property name="text">
         <string>Bank</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="bankLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="codeLabel">
        <property name="text">
         <string>BLZ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="codeLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Schliessen</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Speichern</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>membermanager::gui::SettingsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
